stac-api/src/main/java/io/arlas/server/stac/api/StacConformanceRESTService.java

```diff
@@ -48,14 +50,18 @@ public StacConformanceRESTService(STACConfiguration configuration,
     @Path("/conformance")
     @GET
     @Produces(MediaType.APPLICATION_JSON)
-    @ApiOperation(
-            value = "Conformance",
-            notes = "The URIs of all conformance classes supported by the server.\n" +
-                    "To support \"generic\" clients that want to access multiple OGC API Features implementations - " +
-                    "and not \"just\" a specific API / server, the server declares the conformance classes it implements and conforms to.")
+    @Operation(
+            summary = "Conformance",
+            description = """
+                    The URIs of all conformance classes supported by the server.
+                    To support "generic" clients that want to access multiple OGC API Features implementations -
+                    and not "just" a specific API / server, the server declares the conformance classes it implements and conforms to.""")
     @ApiResponses(value = {
-            @ApiResponse(code = 200, message = "Successful operation", response = ConformanceClasses.class, responseContainer = "ConformanceClasses"),
-            @ApiResponse(code = 500, message = "Arlas Server Error.", response = Error.class)})
+            @ApiResponse(responseCode = "200", description = "Successful operation",
+                    content = @Content(array = @ArraySchema(schema = @Schema(implementation = ConformanceClasses.class)))),
```

Comment:
Returns `ConformanceClasses.class`(not an array)

Reply:
ok